lib/cosmos/tools/tlm_viewer/widgets/labelvalue_widget.rb

```diff
@@ -20,20 +20,38 @@ class LabelvalueWidget < Qt::Widget
     include Widget
     include MultiWidget
 
-    def initialize(parent_layout, target_name, packet_name, item_name, value_type = :WITH_UNITS, characters = 12)
+    def initialize(parent_layout, target_name, packet_name, item_name, value_type = :WITH_UNITS, characters = 12, align = 'center')
       super(target_name, packet_name, item_name, value_type)
       setLayout(Qt::HBoxLayout.new)
       layout.setSpacing(0)
       layout.setContentsMargins(0,0,0,0)
+      layout.addStretch(1) if align.downcase == 'right'
```

Comment:
'center' is the current default behavior but I see what you mean that it's actually centering the space not the values.